src/Database/Migrations/2020-12-28-223112_create_auth_tables.php

```diff
@@ -40,7 +45,7 @@ public function up(): void
         ]);
         $this->forge->addPrimaryKey('id');
         $this->forge->addUniqueKey('username');
-        $this->forge->createTable($this->tables['users']);
+        $this->forge->createTable($this->tables['users'], $this->ifNotExists, $this->attributes);
```

Comment:
This code appears many times. Why don't you move it to a private method?

Reply:
i have moved create table to a private method, so we will get rid of repeated code